terraform/lib/dependabot/terraform/file_parser.rb

```diff
@@ -54,10 +58,10 @@ def parse
 
       private
 
-      def build_terraform_dependency(file, name, details, provider)
+      def build_terraform_dependency(file, name, details)
         details = details.is_a?(Array) ? details.first : details
```

Comment:
When is this an array? Is the same not true for `build_provider_dependency`?

Reply:
I don't think it is anymore, will yank it out 👍